src/nodes.js

```diff
@@ -167,9 +167,7 @@ var getChild = function(parent, key) {
  * @param {!Node} child The child to register.
  */
 var registerChild = function(parent, key, child) {
-  if (key) {
-    getKeyMap(parent)[key] = child;
-  }
+  getKeyMap(parent)[key] = child;
```

Comment:
Didn't even read the comment, updated. I think it should be `{!string}`, though?